config/secrets.yml

```diff
@@ -17,28 +17,19 @@
 
 # Environmental secrets are only available for that specific environment.
 
-default: &base
-  support_provider_key: <%= ENV['SUPPORT_PROVIDER_KEY'] %>
-  support_provider_key_preview: <%= ENV['SUPPORT_PROVIDER_KEY_PREVIEW'] %>
+base: &default
+  secret_key_base: dummy
```

Comment:
I don't understand this change. First, what were support provider keys for? Second, wouldn't be better to set `secret_key_base: <%= ENV['SECRET_KEY_BASE'] %>` as default? why setting `dummy` as default and then overwrite it only for production?

Reply:
We don't really have to set it in fact. This is in [rails](https://github.com/rails/rails/blob/v5.2.7.1/railties/lib/rails/application.rb#L428-L436):
```rb
    def secret_key_base
      if Rails.env.development? || Rails.env.test?
        secrets.secret_key_base ||= generate_development_secret
      else
        validate_secret_key_base(
          ENV["SECRET_KEY_BASE"] || credentials.secret_key_base || secrets.secret_key_base
        )
      end
    end
```

Better to just remove this part.

Reply:
Oh, great point, thanks @akostadinov, I'll remove it.

@jlledom `support_provider_key` was some config that is not used anymore. At some point the swagger specs for 3scale API were stored in "support"'s SaaS account, and the provider key was needed to get those specs. See https://github.com/3scale/system/commit/c3333261bf58699b95087138ba646da585f5ebc8#diff-8dd79c7a3211830fc048071420a263d9f77fa48e0d9d0406b98e61e27cb2d7bf

This is not used anymore.